### ISSUE TITLE
Unknown field 'DisablePointerAddresses' in struct literal of type spew.ConfigState

### ISSUE THREAD

**Issue by bxcodec** [stretchr/testify#574]
It seems your third-party library's `github.com/davecgh/go-spew` have a breaking changes

Got Error
```bash
vendor/github.com/stretchr/testify/assert/assertions.go:1253:25: 
unknown field 'DisablePointerAddresses' in struct literal of type spew.ConfigState
```

**Comment by devdinu** [comment]
Can you share more information or test code to reproduce the issue.

**Comment by bxcodec** [comment]
I'm using glide as my package management. 
I'm trying to update some package, and for some reasons, this package `github.com/davecgh/go-spew`  was updated. 

And after finishing the `glide up`, when running the project's test, I got the error 
```vendor/github.com/stretchr/testify/assert/assertions.go:1253:25: 
unknown field 'DisablePointerAddresses' in struct literal of type spew.ConfigState```

After looking for the source code, it seems the `go-spew`  has a few breaking changes.  

For now, to solve my own issue, I just stay back with the old version of `github.com/davecgh/go-spew`
